src/libraries/Common/tests/System/TimeProviderTests.cs

```diff
@@ -316,12 +316,26 @@ public static void RunDelayTests(TimeProvider provider, ITestTaskFactory taskFac
             Assert.True(task1.Status == TaskStatus.RanToCompletion, "    > FAILED.  Expected Delay(TimeSpan(0), timeProvider) to run to completion");
             Assert.True(task2.Status == TaskStatus.RanToCompletion, "    > FAILED.  Expected Delay(TimeSpan(0), timeProvider, uncanceledToken) to run to completion");
 
+            // This should complete quickly with a CANCELED status.
+            Task task3 = taskFactory.Delay(provider, new TimeSpan(0), new CancellationToken(true));
+            try
+            {
+                var e = Assert.Throws<AggregateException>(task3.Wait);
+                Assert.IsAssignableFrom<OperationCanceledException>(e.InnerException);
+            }
+            catch (Exception e)
+            {
+                Assert.Fail(string.Format("RunDelayTests:    > FAILED.  Unexpected exception on canceled task Wait(): {0}", e));
```

Comment:
nit: use interpolated string `$"..."` instead of string.Format

Reply:
I agree, I was just trying to be consistent with the rest of this test, I wasn't sure what the current guidelines are, I assumed "stay consistent on a per file basis". Would you prefer that I change all the `string.Format` cases in this test? Or the file?

Reply:
Changing it in the file will be ok.

Reply:
Done, it was really only two spots so I feel a bit silly now. 😄